pkg/kudoctl/cmd/install.go

```diff
@@ -31,7 +32,7 @@ var (
 )
 
 // newInstallCmd creates the install command for the CLI
-func newInstallCmd() *cobra.Command {
+func newInstallCmd(fs afero.Fs) *cobra.Command {
```

Comment:
so where are we actually adding the repo flag for the cmdline? 🤔

Reply:
oh ok, I had to jump to a different PR to understand that. So we're adding the flags for all commands? So getting a plan status will have option of setting repo? Or am I missing something?

Reply:
Also can we do `--repo` instead of `--repo-name`?

Reply:
haha... the original code and plan was `--repo` and I changed it to `--repo-name` based on your previous review.  I'll switch back.